package.json

```diff
@@ -24,7 +25,8 @@
     "nan": "^2.14.0"
   },
   "devDependencies": {
-    "jshint": "^2.10.2"
+    "jshint": "^2.10.2",
+    "@types/node": "^12.6.8"
```

Comment:
Why is @types/node a devDependency as it's never used anywhere?

Reply:
This thing is for some NodeJS-only types like `Buffer`. It seems that I need to add this, or otherwise VSCode will show undefined type warnings.

Reply:
For example: 
![image](https://user-images.githubusercontent.com/4463497/61572144-be13b380-aadc-11e9-913a-a6bed81e9b61.png)

Reply:
Ok